lib/cmd/fhc/target.js

```diff
@@ -106,7 +106,13 @@ function target(argv, cb) {
 
       var _gotVersion = function() {
         ini.save(function(err) {
-          target.message = i18n._("Successfully targeted: ") + targ.target + (targ.user ? (" User: " + targ.user) : (" API Key: " + targ[keys.KEY_ID]));
+          var message;
+          if (targ.user) {
+            message = " User: " + targ.user;
+          } else {
+            message = " API Key: " + targ[keys.KEY_ID];
+          }
+          target.message = i18n._("Successfully targeted: " + targ.target + message);
```

Comment:
I don't think it's exactly the same. To be exactly the same, the last line would instead be:

```javascript
target.message = i18n._("Successfully targeted: ") + targ.target + message;
```

I don't know enough about any of this to know which one is more correct. Maybe someone else who knows what's going on here could take a look. I'm confident that the rest of the PR is fine.

@pb82 @wei-lee maybe one of you would be in a better position than I to comment on this?